src/Compilers/CSharp/Portable/Parser/LanguageParser_Patterns.cs

```diff
@@ -485,20 +475,10 @@ private bool ConvertTypeToExpression(TypeSyntax type, [NotNullWhen(true)] out Ex
         private bool LooksLikeTupleArrayType()
         {
             if (this.CurrentToken.Kind != SyntaxKind.OpenParenToken)
-            {
                 return false;
-            }
 
-            ResetPoint resetPoint = GetResetPoint();
```

Comment:
Are there any direct usages of `GetResetPoint` after this?
If all usages have been converted, can we move `GetResetPoint` to be a private method in `DisposableResetPoint` (or inlined, or a local function)?

Reply:
Yes, there are still usages of GetResetPOint.  Commonly for extremely complex cases, like when teh reset point is actually overwritten.